### DISCUSSION TITLE
Any Similar App for Linux?

### DISCUSSION THREAD

**Discussion by ghost** [akanshSirohi/ShareX#21] [Q&A] [answered]
First of all, this is absolutely brilliant! I wanted to take a moment to express my sincerest gratitude for the work you have done. This app is flawless!
I'm impressed by how fast this is, so impressed I would like to ask if there's a similar app in Linux.

I hope I'm not being annoying or asking in the wrong place.
Thanks!

**Comment by akanshSirohi** [comment, marked as answer]
Thanku so much for your feedback!

I don't know about any other app like this for linux 😅

Note: Questions are meant to be asked only 😉
